sentry_sdk/consts.py

```diff
@@ -89,7 +89,6 @@ class ClientConstructor(object):
     def __init__(
         self,
         dsn=None,  # type: Optional[str]
-        with_locals=True,  # type: bool
```

Comment:
removing something from here is a breaking change. This we have to do in a major.

I would keep this here and if present, set the other one in client.py

Reply:
The users can still use `with_locals` in `sentry_sdk.init()` it will just be then copied to the new var name here: https://github.com/getsentry/sentry-python/blob/antonpirker/1854-include-local-variables/sentry_sdk/client.py#L74-L82